src-electron/db/query-feature.js

```diff
@@ -0,0 +1,76 @@
+/**
+ *
+ *    Copyright (c) 2021 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * This module provides queries for features.
+ *
+ * @module DB API: feature related queries
+ */
+const dbApi = require('./db-api.js')
+const dbMapping = require('./db-mapping.js')
+
+/**
+ * Get all device type features associated with a list of device type refs
+ * @param {*} db
+ * @param {*} deviceTypeRefs
+ * @returns All feature information and device type conformance
+ * with associated device type and cluster details
+ */
+async function getFeaturesByDeviceTypeRefs(db, deviceTypeRefs) {
```

Comment:
Instead of creating a separate file for this. This api can go into the query-device-type.js

Reply:
This would have to be reorganized if we added "cluster level" features in the future OR if there was a new spec with features defined in another way. It would make more sense to have a new file because it would be dynamic and not tied to device types. No?

Reply:
query-feature is going to have more functions in the future, and just as q uery-attribute/command, I think we should have a separate file for this